contracts/governance/AxelarServiceGovernance.sol

```diff
@@ -0,0 +1,101 @@
+     * @param signers The list of initial signers
+     * @param threshold The number of required signers to validate a transaction
+     */
+    constructor(
+        address gateway,
+        string memory governanceChain,
+        string memory governanceAddress,
+        uint256 minimumTimeDelay,
+        address[] memory signers,
+        uint256 threshold
+    ) InterchainGovernance(gateway, governanceChain, governanceAddress, minimumTimeDelay) MultisigBase(signers, threshold) {}
+
+    /**
+     * @notice Executes a multisig proposal.
+     * @param target The target address the proposal will call
+     * @param callData The data that encodes the function and arguments to call on the target contract
+     */
+    function executeMultisigProposal(
+        address target,
+        bytes calldata callData,
+        uint256 value
+    ) external payable onlySigners {
+        bytes32 proposalHash = keccak256(abi.encodePacked(target, callData, value));
+
+        if (!multisigApprovals[proposalHash]) revert NotApproved();
+
+        multisigApprovals[proposalHash] = false;
+
+        _call(target, callData, value);
+
+        emit MultisigExecuted(proposalHash, target, callData, value);
+    }
+
+    /**
+     * @notice Internal function to process a governance command
+     * @param commandId The id of the command
+     * @param target The target address the proposal will call
+     * @param callData The data the encodes the function and arguments to call on the target contract
+     * @param nativeValue The value of native token to be sent to the target contract
+     * @param eta The time after which the proposal can be executed
+     */
+    function _processCommand(
+        uint256 commandId,
+        address target,
+        bytes memory callData,
+        uint256 nativeValue,
+        uint256 eta
+    ) internal override {
+        ServiceGovernanceCommand command = ServiceGovernanceCommand(commandId);
+        bytes32 proposalHash = keccak256(abi.encodePacked(target, callData, nativeValue));
+
+        if (command == ServiceGovernanceCommand.ScheduleTimeLockProposal) {
+            eta = _scheduleTimeLock(proposalHash, eta);
+
+            emit ProposalScheduled(proposalHash, target, callData, nativeValue, eta);
+        } else if (command == ServiceGovernanceCommand.CancelTimeLockProposal) {
+            _cancelTimeLock(proposalHash);
+
+            emit ProposalCancelled(proposalHash, target, callData, nativeValue, eta);
+        } else if (command == ServiceGovernanceCommand.ApproveMultisigProposal) {
+            multisigApprovals[proposalHash] = true;
+
+            emit MultisigApproved(proposalHash, target, callData, nativeValue);
+        } else if (command == ServiceGovernanceCommand.CancelMultisigApproval) {
+            multisigApprovals[proposalHash] = false;
+
+            emit MultisigCancelled(proposalHash, target, callData, nativeValue);
+        } else {
+            revert InvalidCommand();
```

Comment:
since this can't be reached, would it make sense to add following check at the top instead (and similarly for the other governance contract) instead of letting it panic implicitly
```
if (commandId > type(ServiceGovernanceCommand).max) {
  revert ...
}
```